cloudbuild.yaml

```diff
@@ -27,7 +27,7 @@ steps:
     args:
       - importer-image-push
     env:
-      - IMAGE_REGISTRY=gcr.io/$PROJECT_ID
+      - STAGING_IMAGE_REGISTRY=gcr.io/$PROJECT_ID
```

Comment:
The Makefile needs to be changed to accept this as a value. It's currently a constant

Reply:
done